lib/timer_page.dart

```diff
@@ -113,62 +130,40 @@ class TimerTextState extends State<TimerText> {
     return new Row(
       mainAxisAlignment: MainAxisAlignment.center,
       children: <Widget>[
-        new Expanded(
-          flex: 2,
-          child: new RepaintBoundary(
+          new RepaintBoundary(
             child: new SizedBox(
               height: 72.0,
-              child: new MinutesAndSeconds(stopwatch),
+              child: new MinutesAndSeconds(dependencies: dependencies),
             ),
           ),
-        ),
-        new Expanded(
-          flex: 1,
-          child: new RepaintBoundary(
+          new RepaintBoundary(
             child: new SizedBox(
               height: 72.0,
-              child: new Hundreds(stopwatch),
+              child: new Hundreds(dependencies: dependencies),
             ),
           ),
-        ),
```

Comment:
Removed `Expanded` widgets: layout looks the same without.

Reply:
Are you sure it's the same? The reason I used `Expanded` was to make sure that `MinutesAndSeconds` get 2/3 of the screen and `Hundreds` gets 1/3 of the screen. Without it, when you start the clock the dot jumps around the screen a lot.

Reply:
Ideally though, the layout would be such that the text doesn't shift around. A monospace font could do that. Or the clock could be split further into Minutes, Seconds and Hundreds, such that each of them has a fixed width.

Reply:
The numerals for the font I'm using (Open Sans) all have the same width. When I test it all text stays in place.

Reply:
The width of digits changes when I run the app on iOS, but it's fixed when running on Android. I think what's happening is that "Open Sans" is mapped to different system fonts (the two OSes have different sets of preinstalled fonts). On Android it happens to map to one with fixed-width digits. Not sure what it maps to on iOS. To get consistent behavior you might want to bundle your own font: https://flutter.io/custom-fonts/

Reply:
Ok. I'll look at fonts separately.